synapse/lib/stormlib/env.py

```diff
@@ -47,4 +46,4 @@ async def _libEnvGet(self, name, default=None):
             mesg = f'Environment variable must start with SYN_STORM_ENV_ : {name}'
             raise s_exc.BadArg(mesg=mesg)
 
-        return os.getenv(name, default=default)
+        return os.getenv(name, default=await s_stormtypes.tostr(default, noneok=True))
```

Comment:
We should replace the default value above on line 43 with the tostr() call.

Reply:
that's icky and frustrating; and likely dependent upon how an object was created :(

Reply:
Yeah using toprim -> tostr seems like it would provide the best results to me.